frontend/src/components/Patchnotes.jsx

```diff
@@ -75,7 +91,21 @@ const Patchnotes = ({ game }) => {
       }
     };
 
+    const fetchAssociations = async () => {
+      try {
+        const response = await fetch('http://localhost:3000/associations');
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
```

Comment:
specific error messages: "Network response when fetching associations was not ok" can go a long way when you run into more problems later